### ISSUE TITLE
Ability to init with raw command instead of flags

### ISSUE THREAD

**Issue by AkisArou** [dmmulroy/tsc.nvim#45]
I have a monorepo where I use ts project references. I want to start the tsc from the nvim cwd (repo root dir), by executing "tsc --build /home/user/my-repo/tsconfig.json --watch".

I investigated the code, and I could not figure out a way to do this (specifically to define the path of tsconfig).
Project references build command cannot be invoked with --project.
I tried to trick it with bin_path option, but you check whether is executable.
Via the flags you prepend double-dash.

Maybe you can provide a way to directly specify the command instead of the various options-flags?
I think this may be the only way, because project references --build and rest args, should be placed in a specific order. (--build first, then tsconfig, then rest args).

Thanks!
Nice plugin btw :)

**Comment by dmmulroy** [comment]
If the plugin respected the order of the flags in the config would that resolve this?

**Comment by AkisArou** [comment]
I do not think so because you prepend -- at flags and in the command for ts project references, the arg for project tsconfig has none. 
I find it difficult to abstract away flags-options and args to cover all cases. That's why I proposed a raw command

**Comment by benfc1993** [comment]
Is the main issue that you want to run the `TSC` usrcmd and specify the path to the dir to use? 
If so would adding an option to specify a dir when running the `TSC` command work for you? Just thinking providing an option to hard code it in the package config would mean editing your nvim config whenever you change what you are working on. 

The other option would maybe be the ability to provide a function which resolves to a string which can be run to set the location of the config when `TSC` is called?

**Comment by AkisArou** [comment]
Yes I think this will resolve it! 
This will contradict the `project` flag.
Setting `project = false` so no default flag is used, will be needed.
